tomobar/methodsDIR_CuPy.py

```
@@ -214,7 +269,9 @@ def FOURIER_INV(self, data: xp.ndarray, **kwargs) -> xp.ndarray:
                     chunk_count = value
             if key == "slice_count_per_chunk" and value is not None:
                 if slice_count_per_chunk is not int or value <= 0:
-                    print(f"Invalid slice count: {value}. Set to slice count of input data")
+                    print(
```

Should we raise an exception here instead of print?